### ISSUE TITLE
Linting rules inconsistent with nomad-lab

### ISSUE THREAD

**Issue by lauri-codes** [FAIRmat-NFDI/cookiecutter-nomad-plugin#14]
The linting rules for plugins are not currently identical with the ones we use for the nomad-lab package (https://gitlab.mpcdf.mpg.de/nomad-lab/nomad-FAIR/-/blob/2002-move-porositynormalizer-to-a-plugin/pyproject.toml). Specifically we are ignoring more stuff in nomad-lab than in plugins.

Is there a specific reason for this, or could we make these two compatible?

**Comment by blueraft** [comment]
A lot of the rules that we ignore are generally standard rules and would have been good to include them in `nomad-lab` too but it was quite  `nomad-lab` difficult to fix those violations in such a big existing code base.

In this template, if the user starts off from a fresh code base, then it becomes much easier avoid those violations. Or if necessary, include the `# noqa` block for a specific violation. 

You're probably trying to migrate code from the `nomad-lab` code base to a plugin which is causing these errors I presume? Do you have any specific code rules in mind that we should ignore in general?

**Comment by lauri-codes** [comment]
Makes sense. So I guess the correct way is to instead make the `nomad-lab` linting rules more consistent with the plugin rules.

As you said, I think we should probably adhere to the defaults as best as we can and use `noqa` when deemed necessary. Closing for now.
